doc/modules/multiclass.rst

```diff
@@ -87,18 +87,26 @@ zero elements, corresponds to the subset of labels. An array such as
 ``np.array([[1, 0, 0], [0, 1, 1], [0, 0, 0]])`` represents label 0 in the first
 sample, labels 1 and 2 in the second sample, and no labels in the third sample.
 
-Producing multilabel data as a list of sets of labels may be more intuitive.
```

Comment:
This statement was inserted as part of the deprecation, and should stay. It makes the motivation for `MultiLabelBinarizer` clearer.